examples/composite_dialog/my_app_window/imp.rs

```diff
@@ -0,0 +1,55 @@
+use std::cell::Cell;
+
+use gtk::{
+    glib::{self, Properties},
+    prelude::*,
+    subclass::prelude::*,
+    CompositeTemplate,
+};
+
+// Object holding the state
+#[derive(Default, CompositeTemplate, Properties)]
```

Comment:
```suggestion
#[derive(Default, gtk::CompositeTemplate, glib::Properties)]
```
No need for includes for type used only once.